From 120915d6deff05c0154c9d44c3903a8a6e1b3036 Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Thu, 23 Dec 2010 16:58:26 +0000
Subject: Reinstate the implementation of gdk_window_foreign_new_for_display

This was accidentally dropped when introducing
gdk_x11_window_foreign_new_for_display().
---
diff --git a/gdk/x11/gdkwindow-x11.c b/gdk/x11/gdkwindow-x11.c
index d84e0e0..c2c32f0 100644
--- a/gdk/x11/gdkwindow-x11.c
+++ b/gdk/x11/gdkwindow-x11.c
@@ -880,6 +880,7 @@ GdkWindow *
 gdk_window_foreign_new_for_display (GdkDisplay     *display,
 				    GdkNativeWindow anid)
 {
+  return gdk_x11_window_foreign_new_for_display (display, anid);
 }
 
 /**
--
cgit v0.8.3.1
